src/cli/repl.rs

```diff
@@ -543,24 +540,22 @@ impl Repl<F> {
                         Ok((.., expr_ptr, is_meta)) => {
                             if is_meta {
                                 if let Err(e) = self.handle_meta(expr_ptr) {
-                                    println!("!Error: {e}");
+                                    println!("!Error: {e}")
                                 }
                             } else if let Err(e) = self.handle_non_meta(expr_ptr) {
-                                println!("Error: {e}");
+                                println!("Error: {e}")
                             }
                         }
                         Err(parser::Error::NoInput) => (),
-                        Err(e) => {
-                            println!("Read error: {e}")
-                        }
+                        Err(e) => println!("Read error: {e}"),
                     }
                 }
                 Err(ReadlineError::Interrupted | ReadlineError::Eof) => {
                     println!("Exiting...");
                     break;
                 }
-                Err(err) => {
-                    println!("Read line error: {err}");
+                Err(e) => {
+                    println!("Read line error: {e}");
```

Comment:
Those cases are pretty nearly the definition of when `eprintln!` is more appropriate than `println!`.